posts/2023-05-31-adding-new-task-to-cylc.md

```diff
@@ -0,0 +1,150 @@
+# Adding a new task to a rose suite
+
+## Issue
+In this example, we want to add an additional task to an ACCESS-CM2 run, to be executed before every instance of the coupler to update the sea surface temperature ancillary file.
```

Comment:
I would probably twist this around so saying first what you want to achieve and then what you need to do, i.e.

We want to update the sea surface temperature ancillary file at each ...so we need to add an additional ... to be execute before every instance of the coupler.